src/riak_cs_console.erl

```diff
@@ -183,10 +185,119 @@ cleanup_orphan_multipart(Timestamp) when is_binary(Timestamp) ->
     _ = io:format("~nAll old unaborted orphan multipart uploads have been deleted.~n", []).
 
 
+-spec resolve_siblings(binary(), binary()) -> any().
+resolve_siblings(RawBucket, RawKey) ->
+    {ok, RcPid} = riak_cs_riak_client:start_link([]),
+    try
+        {ok, Pid} = case RawBucket of
+                        <<"0o:", _/binary>> ->
+                            riak_cs_riak_client:manifest_pbc(RcPid);
+                        <<"0b:", _/binary>> ->
+                            riak_cs_riak_client:block_pbc(RcPid);
+                        _ ->
+                            riak_cs_riak_client:master_pbc(RcPid)
+                    end,
+        resolve_siblings(Pid, RawBucket, RawKey)
+    after
+        riak_cs_riak_client:stop(RcPid)
+    end.
+
+%% @doc Resolve siblings as quick operation. Assuming login via
+%% `riak-cs attach` RawKey and RawBucket should be provided via Riak
+%% log. Example usage is:
+%%
+%% > {ok, Pid} = riakc_pb_socket:start_link(Host, Port).
+%% > riak_cs_console:resolve_siblings(Pid, <<...>>, <<...>>).
+%% > riakc_pb_socket:stop(Pid).
+%%
+%% or, in case of manifests, for CS Bucket and Key name:
+%%
+%% > {ok, Pid} = riakc_pb_socket:start_link(Host, Port).
+%% > RawBucket = riak_cs_utils:to_bucket_name(objects, CSBucketName),
+%% > riak_cs_console:resolve_siblings(Pid, RawBucket, CSKey).
+%% > riakc_pb_socket:stop(Pid).
+%%
+-spec resolve_siblings(pid(), RawBucket::binary(), RawKey::binary()) -> ok | {error, term()}.
+resolve_siblings(Pid, RawBucket, RawKey) ->
+    GetOptions = [{r, all}],
+    PutOptions = [{w, all}],
+    resolve_siblings(Pid, RawBucket, RawKey,
+                     GetOptions, ?DEFAULT_RIAK_TIMEOUT,
+                     PutOptions, ?DEFAULT_RIAK_TIMEOUT).
+
+resolve_siblings(Pid, RawBucket, RawKey, GetOptions, GetTimeout, PutOptions, PutTimeout)
+  when is_integer(GetTimeout) andalso is_integer(PutTimeout) ->
+    _ = lager:info("Getting ~p:~p~n", [RawBucket, RawKey]),
+    case riakc_pb_socket:get(Pid, RawBucket, RawKey, GetOptions, GetTimeout) of
+        {ok, RiakObj} ->
+            _ = lager:info("Trying to resolve ~p sibling(s) of ~p:~p",
+                           [riakc_obj:value_count(RiakObj), RawBucket, RawKey]),
+            case resolve_ro_siblings(RiakObj, RawBucket, RawKey) of
+                {ok, RiakObj2} ->
+                    R = riakc_pb_socket:put(Pid, RiakObj2, PutOptions, PutTimeout),
+                    _ = lager:info("Resolution result: ~p~n", [R]),
+                    R;
+                {error, R} = E ->
+                    _ = lager:info("Not updating ~p:~p: ~p~n",
+                                   [RawBucket, RawKey, R]),
```

Comment:
I don't think so, if that's error it should have log level as error but this is manual operation tool so I assume the operator's watching the log and see it. If needed, it should be grep'd with `Not updating` .